kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -16,19 +16,19 @@ data:
           # In the absense of this configmap, default value for enabled is true
           enabled = true
           # exclude_namespaces setting holds good only if enabled is set to true
-          # kube-system log collection is disabled by default in the absence of 'log_collection_settings.stdout' setting. If you want to enable kube-system, remove it from the following setting.
-          # If you want to continue to disable kube-system log collection keep this namespace in the following setting and add any other namespace you want to disable log collection to the array.
-          # In the absense of this configmap, default value for exclude_namespaces = ["kube-system"]
-          exclude_namespaces = ["kube-system"]
+          # kube-system, gatekeeper-system log collection are disabled by default in the absence of 'log_collection_settings.stdout' setting. If you want to enable kube-system, gatekeeper-system, remove them from the following setting.
+          # If you want to continue to disable kube-system, gatekeeper-system log collection keep the namespaces in the following setting and add any other namespace you want to disable log collection to the array.
+          # In the absense of this configmap, default value for exclude_namespaces = ["kube-system, gatekeeper-system"]
+          exclude_namespaces = ["kube-system, gatekeeper-system"]
```

Comment:
isn't this to be ["kube-system", "gatekeeper-system"] ?

Reply:
I see changed